client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -227,10 +227,7 @@ class SiteImporterInputPane extends React.Component {
 			site_url: urlForImport,
 		} );
 
-		loadmShotsPreview( {
-			url: urlForImport,
-			maxRetries: 1,
```

Comment:
This existing usage has it retrying one time -- is that desirable?

Reply:
Looks like the e2e test isn't passing d/t this change.  Investigating further.

Reply:
Could it be failing because the `catch` is now unhandled (even though before it was just a `noop`)?

Reply:
Very possible, but I thought this fn defaulted to noop'ing

Reply:
Yah you're right! https://github.com/Automattic/wp-calypso/pull/28131/files#diff-9b00647c8a9f40270e3f1abfa94367d5R15